ADR/0032-decoupling-deployment.md

```diff
@@ -0,0 +1,201 @@
+# 32. Decoupling Deployment
+
+Date: 2023-11-17
+
+## Status
+
+Draft
+
+## Authors
+
+- Ralph Bean
+
+## Context
```

Comment:
This affects multiple other ADRs. Can we establish the links to ADRs that would change based on this? I was thinking something similar to what I did here: https://github.com/redhat-appstudio/book/pull/103/files -- mentioning what is deprecated (or partially deprecated) and what the relationship links are.

It would be helpful to consider the full impact of this ADR.

Reply:
Done in cc25057.